src/sentry/static/sentry/app/components/todos.jsx

```diff
@@ -0,0 +1,314 @@
+      classNames += ' checked';
+
+      if (this.props.task.task == 0) {
+        description = doneDescription;
+      } else {
+        description = this.props.task.user + ' completed this ' + moment(this.props.task.date_completed).fromNow();
+      }
+
+    } else if (this.props.task['status'] == 'Pending') {
+      classNames += ' pending';
+      description = this.props.task.user + ' kicked this off ' + moment(this.props.task.date_completed).fromNow();
+    } else if (this.props.task['status'] == 'Skipped') {
+      classNames += ' skipped';
+      description = this.props.task.user + ' skipped this ' + moment(this.props.task.date_completed).fromNow();
+    } else {
+      description = this.props.task.description;
+    }
+
+    if (this.state.showConfirmation) {
+      classNames += ' blur';
+    }
+
+    var learn_more_url= '';
+    if (this.props.task['feature_location'] === 'project') {
+      learn_more_url = '/organizations/' + org.slug + '/projects/choose/?next=' + this.props.task['location'];
+    } else if (this.props.task['feature_location'] === 'organization') {
+      learn_more_url = '/organizations/' + org.slug + '/' + this.props.task['location'];
+    } else if (this.props.task['feature_location'] === 'absolute') {
+      learn_more_url = this.props.task['location'];
+    }
+
+    return (
+      <li className={classNames}>
+        { this.props.task['status'] == 'Pending' ? <span className="pending-bar" /> : null }
+        <div className="todo-content">
+          <div className="ob-checkbox">
+            { this.props.task['status'] == 'Complete' ? <span className="icon-checkmark" /> : null }
+            { this.props.task['status'] == 'Skipped' ? <span className="icon-x" /> : null }
+            { this.props.task['status'] == 'Pending' ? <span className="icon-ellipsis" /> : null }
+          </div>
+          <a href={learn_more_url}><h4>{ this.props.task['title'] }</h4></a>
+          <p>
+            { description }
+          </p>
+          { this.props.task['skippable'] && this.props.task['status'] != 'Skipped' && this.props.task['status'] != 'Complete' && !this.state.showConfirmation ?
+            <a className="skip-btn btn btn-default" onClick={this.toggleConfirmation}>Skip</a> : null }
+        </div>
+        { this.state.showConfirmation ? <Confirmation task={this.props.task['task']} onSkip={this.skip} dismiss={this.toggleConfirmation} /> : null }
+      </li>
+    );
+  }
+});
+
+const Confirmation = React.createClass({
+  propTypes: {
+    task: React.PropTypes.string,
+  },
+
+  skip: function(e) {
+    e.preventDefault();
+    this.props.onSkip(this.props.task);
+  },
+
+  dismiss: function() {
+    this.props.dismiss();
+  },
+
+  render: function() {
+    return (
+      <div className="ob-confirmation" onClick={this.dismiss}>
+        <h3>Need help?</h3>
+        <p><a href="mailto:support@getsentry.com?subject=Help with onboarding">Ask us!</a> &middot; <a onClick={this.skip}>Skip</a></p>
```

Comment:
This support email will be problematic for on-premise/oss users. We wouldn't want them emailing support@ blindly.